packages/sdk/src/interaction/interaction.js

```diff
@@ -106,9 +107,75 @@ const IX = `{
   }
 }`
 
+const LOGGER_LEVELS = {
+  "debug": 0,
+  "info": 1,
+  "log": 2,
+  "warn": 3,
+  "error": 4
+}
+
+const logger = async (title, message, level) => {
```

Comment:
Can we move this elsewhere?

Reply:
Yeah I also don't like it there. It needs its own package one day. Maybe root SDK for now?

Reply:
maybe utils/

Reply:
There's some deprecation stuff started in there